api/client/client.go

```diff
@@ -4317,6 +4317,20 @@ func (c *Client) UpsertCertAuthority(ctx context.Context, ca types.CertAuthority
 	return out, trace.Wrap(err)
 }
 
+// RotateExternalCertAuthority rotates the provided cert authority.
+func (c *Client) RotateExternalCertAuthority(ctx context.Context, ca types.CertAuthority) error {
+	cav2, ok := ca.(*types.CertAuthorityV2)
+	if !ok {
+		return trace.BadParameter("unexpected ca type %T", ca)
+	}
+
+	_, err := c.TrustClient().RotateExternalCertAuthority(ctx, &trustpb.RotateExternalCertAuthorityRequest{
```

Comment:
Could we ditch the boilerplate and move callers to TrustClient() directly?

Reply:
This boilerplate seems genuinely useful from the consumer perspective of Client. It would be a pain to parse through each underlying client to find what you're looking for. It's also in line with other similar client implementations and serves to implement existing interfaces.